src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,6 +14,9 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int sequenceNumber;
+    private static int initNextSequenceNumber = 1;
```

Comment:
this is not adding any additional information/value to the code, hence can be removed.